### ISSUE TITLE
Window not Outside

### ISSUE THREAD

**Issue by lyxal** [Random-People/Chit-chat#6]
Getting a `SkyDoesn'tExist` error when trying to check the weather. Only happens when looking through the window.

**Comment by Guigui220D** [comment]
Are you sure you're not already under the sky when looking through the window? You can't look at the sky through a window if you're at the wrong side of the window. Maybe you should revert to when you were inside, or try not to look through the window? Try using the terminal instead of a gui, so you won't need a window

**Comment by lyxal** [comment]
I looked up and saw roof. Window seems to be inside.

**Comment by Guigui220D** [comment]
You're probably looking at the wrong kind of window. Is it a microwave?

**Comment by ghost** [comment]
How can I install sway on my microwave please? I'm interested in intensive tiling for separate cooking.

**Comment by lyxal** [comment]
> You're probably looking at the wrong kind of window. Is it a microwave?

No, it is definitely a proper window. I look through it and see wall instead of sky.
